index.js

```diff
@@ -269,11 +265,11 @@ resolve.sync = function (id, opts) {
 
     opts = opts || {};
     var base = path.dirname(opts.filename);
-    
+
     if (opts.basedir) {
         base = opts.basedir;
     }
-   
+
```

Comment:
yea, my sublime is set to strip spaces on save. It's not obvious in the diff, but it's a removal